chore: add kb for binding radiogroup to enum (#1501)
telerik/blazor-docs: knowledge-base/radiogroup-bind-to-enum.md

---
title: Bind RadioGroup to an Enum
description: Is there a way to bind the Data source for a Blazor UI RadioGroup to an Enum?
type: how-to
page_title: Bind RadioGroup to an Enum
slug: radiogroup-kb-enum-binding
position: 
tags: radiogroup, binding, enum
ticketid: 1557229
res_type: kb
---

## Environment

<table>
    <tbody>
        <tr>
            <td>Product</td>
            <td>RadioGroup for Blazor</td>
        </tr>
    </tbody>
</table>

## Description

> Is there a way to bind the Data source for a Blazor UI RadioGroup to an Enum?

Reviewer: ```suggestion

Is there a way to bind the data source for a Blazor UI RadioGroup to an enum?
```

## Solution

> You need to prepare a list of all the available 

Reviewer: ```suggestion
```

> You need to prepare a list of items that correspond to the enum values that can be shown to the user. Here is an example of this:

Reviewer: ```suggestion
To achieve this, prepare a list of items that correspond to the enum values that can be shown to the user. Here is an example:
```
